static/js/components/NewShift.jsx

```diff
@@ -12,6 +12,11 @@ import { fetchShifts } from "../ducks/shifts";
 
 dayjs.extend(utc);
 
+function isDailyShift(shiftName) {
+  const regex = /\d+\/\d+/;
+  return regex.test(shiftName);
```

Comment:
```suggestion
function isDailyShift(shiftName) {
  const regex = /\d+\/\d+$/;
  return regex.test(shiftName);
```
The added `$` at the end makes sure it only matches the pattern at the end of the string.